x/conflict/README.md

```diff
@@ -0,0 +1,124 @@
+# `x/conflict`
+
+## Abstract
+
+This document specifies the conflict module of Lava Protocol.
+
+The purpose of the conflict module is to ensure data reliability from providers that offer their services on Lava. To achieve this, Lava allows consumers to report mismatches between different responses to the same request (for deterministic APIs). This way, Lava can identify fraudulent providers and penalize them by confiscating their staked tokens. Consumers are incentivized to do so by Lava, as they receive tokens taken from the fraudulent provider being punished.
+
+
+## Contents
+* [Concepts](#concepts)
+    * [Finalization Conflict](#finalization-conflict)
+    * [Self Provider Conflict](#self-provider-conflict)
+    * [Response Conflict](#response-conflict)
+    * [Commit Period](#commit-period)
+    * [Reveal Period](#reveal-period)
+    * [Conflict Resolve](#Conflict-Resolve)
+* [Parameters](#parameters)
+* [Queries](#queries)
+* [Transactions](#transactions)
+* [Events](#events)
+
+## Concepts
+
+### Finalization Conflict
+N/A
+
+
+### Self Provider Conflict
+N/A
+
+
+### Response Conflict
+A response conflict occurs when a consumer receives mismatched responses from different providers. In such cases, the consumer is eligible to send a conflict detection message, which includes the relay request and responses from the two providers. This conflict detection message is then validated to ensure that the responses are different, the signatures match the providers and consumer, and that the API is deterministic. If the message is valid, a conflict is opened.
+
+A group of validators is selected as a jury to determine the fraudulent and honest providers. Through an event, the chain announces the conflict voting period and the participating providers. During the voting period, providers need to submit their hashed response + salt to the original relay request. This is done to prevent other providers from cheating or copying their vote. Once the voting period ends, the conflict moves to the reveal state. In this state, providers need to reveal their response + salt, which is then verified and compared to the original responses. After the reveal period ends, the votes are counted, and the provider with the fewest votes is penalized by having a fraction of their staked tokens taken and distributed among all the participants.
```

Comment:
regarding the penalty for losing the conflict, don't also the ones that voted for the losing provider get penalized too?

Reply:
yes they are